LogProcessing.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Mar 15 17:07:04 2019
+
+@author: Nataliya_Pavych
+"""
+
+
+def load_file_into_list_by_line(file_name: str) -> object:
+    with open(file_name) as text_file:
+        return [line for line in text_file]
+
+
+def clean_data_into_d2_list(list_name):
+    return list(map(lambda string: (lambda x, y, z: [f'{x} {y[1:12]}', int(z)])(*string.split(' ')), list_name))
+
+
+def get_hours_from(list_name):
+    return list(map(lambda string: string.split(':')[1], list_name))
+
+
+def sum_values_by_key_into_dictionary(list_name):
+    dictionary = {}
+    for element in list_name:
+        if element[0] in dictionary:
+            dictionary.update({element[0]: dictionary[element[0]]+element[1]})
```

Comment:
dict.update is useful when updating a dict with other dict's multiple values.
In case if this is a spot update, just use
dictionary[key] = new_value

Reply:
`def sum_values_by_key_into_dictionary(list_name):
    dictionary = {}
    for element in list_name:
        if element[0] in dictionary:
            dictionary[element[0]] = dictionary[element[0]]+element[1]
        else:
            dictionary[element[0]] = element[1]
    return dictionary`